### PR TITLE
[DSY-911] Feat/add typography tokens

### PR DESCRIPTION
**Pull request by arielwb**
# Description

This PR adds the `typography` tokens and also does a full refactoring of the `ios` output format

127 0 0 1_8080_avon_light html

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Unit test
- [x] 100% coverage

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/natds-themes/config/android/__file_snapshots__/registerAttributesFormat.snap.xml [hunk 1]
```diff
@@ -21,10 +21,16 @@
 
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <attr name="colorPrimary" format="color" />
-
-    <attr name="sizeTiny" format="dimension" />
-
-    <attr name="spacingSmall" format="dimension" />
-
+  <attr name="" format="" />
```
Comment (jimmyandrade) [inline packages/natds-themes/config/android/__file_snapshots__/registerAttributesFormat.snap.xml:24]: Curiosidade: é para estes atributos  ficarem vazios mesmo?

File: package.json [hunk 2]
```diff
@@ -41,7 +41,9 @@
     "@commitlint/cli": "9.1.1",
     "@commitlint/config-conventional": "9.1.1",
     "@naturacosmeticos/eslint-config-natura": "2.0.2",
+    "babel-eslint": "^10.1.0",
```
[the remark this answers is not in the document]
Comment (arielwb) [reply package.json]: boa! 18c9d6173fa3dceaf14dfa172428e5f5d5aa76c0

File: packages/natds-themes/build/html/avon/dark.html [hunk 3]
```diff
@@ -6,10 +6,16 @@
...
   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
     integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+
 </head>
 
 <body>
   <style>
+    * {
+      font-family: 'Roboto';
```
[the remark this answers is not in the document]
Comment (arielwb) [reply packages/natds-themes/build/html/avon/dark.html]: done! 18c9d6173fa3dceaf14dfa172428e5f5d5aa76c0

File: packages/natds-themes/config/android/__file_snapshots__/registerAttributesFormat.snap.xml (hunk 1 above)
Comment (arielwb) [reply packages/natds-themes/config/android/__file_snapshots__/registerAttributesFormat.snap.xml]: não! foi um ótimo comentario, gracias ;)
18c9d6173fa3dceaf14dfa172428e5f5d5aa76c0

File: packages/natds-themes/build/html/avon/light.html [hunk 4]
```diff
@@ -13,7 +13,7 @@
 <body>
   <style>
     * {
-      font-family: 'Roboto';
+      font-family: 'Roboto, sans-serif';
```
Comment (jimmyandrade) [inline packages/natds-themes/build/html/avon/light.html:16]: @arielwb idealmente, deveria ficar assim: `font-family: Roboto, sans-serif`, sem as aspas, senão o processador de CSS pode entender como um nome de fonte só. Se for estritamente necessário usar aspas, você poderia usar assim: `font-family: 'Roboto', 'sans-serif';`.

Comment (arielwb) [reply packages/natds-themes/build/html/avon/light.html]: listo ;)

**Pull request merged**